### PR TITLE
Fix card widths on initial load

### LINKED ISSUE #957: Fix market card width on initial page load
market card goes to 100% width on horizontal scroll components during initial load

### PR DESCRIPTION
**Pull request by robhyrk**
refactored to use CSS breakpoints for calculating card widths

closes #957

### CODE REVIEW

File: components/markets/MarketAssetDetails.tsx [hunk 1]
```diff
@@ -129,6 +129,7 @@ const MarketAssetDetails = observer(
                     )
                   : null,
               change: priceChange,
+              poolId: market.pool.poolId,
```
Comment (Robiquet) [inline components/markets/MarketAssetDetails.tsx:132]: Is this needed?

File: components/ui/Table.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ import Avatar from "./Avatar";
 import { range } from "lodash";
 import { useIsOnScreen } from "lib/hooks/useIsOnScreen";
 import { useZtgInfo } from "lib/hooks/queries/useZtgInfo";
+import { getCurrentPrediction } from "lib/util/assets";
```
Comment (Robiquet) [inline components/ui/Table.tsx:17]: Think this can be removed too

**Review by Robiquet — COMMENTED** [review]
Looks good, just need to remove the unused code

File: components/markets/MarketAssetDetails.tsx (hunk 1 above)
Comment (robhyrk) [reply components/markets/MarketAssetDetails.tsx]: Accidentally popped some stashed code here. Will remove.

**Review by TvrtkoM — APPROVED** [review]

**Pull request merged**